conan/tools/microsoft/visual.py

```diff
@@ -177,7 +177,7 @@ def msvc_runtime_flag(conanfile):
     Gets the MSVC runtime flag given the ``compiler.runtime`` value from the settings.
 
     :param conanfile: ``< ConanFile object >`` The current recipe object. Always use ``self``.
-    :return: ``str`` runtime flag.
+    :return: ``str`` runtime flag or ``None``.
```

Comment:
I am concerned that this is breaking existing usages of this function, as this is exposed as a tool. I'd prefer to keep the current behavior returning ``""`` empty string.

Reply:
Ah OK; done.